recipes/taxoniq/ncbi_refseq_accession_db/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "ncbi_refseq_accession_db" %}
```

Comment:
These recipes should not be nested.

You also removed the "extras" sections which should list you as a maintainer. Please copy and adapt the section accordingly from the example.

Reply:
Hi,
Thanks, I will add the recipes separately and add the missing sections.